components/server/src/auth/auth-provider.ts

```diff
@@ -65,6 +65,7 @@ export interface AuthUser {
     readonly authId: string;
     readonly authName: string;
     readonly primaryEmail: string;
+    readonly additionalEmails?: { address: string, type: string }[];
```

Comment:
nit: could be typed as `Email` as well, right?

Reply:
👍  Yep! I believe this is now resolved.